google/resource-snippets/cloudkms-v1/alternatives/krm/Kptfile

```diff
@@ -0,0 +1,23 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: cloudkms
+packageMetadata:
+  shortDescription: configure cloudkms
```

Comment:
Nit: How about something more descriptive like "Example Kpt package for configuring a Cloud KMS key"